js/modules/k6/webcrypto/tests/wpt-patches/WebCryptoAPI__derive_bits_keys__ecdh_bits.js.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/WebCryptoAPI/derive_bits_keys/ecdh_bits.js b/WebCryptoAPI/derive_bits_keys/ecdh_bits.js
+index 36b29c20a..c9c413599 100644
+--- a/WebCryptoAPI/derive_bits_keys/ecdh_bits.js
++++ b/WebCryptoAPI/derive_bits_keys/ecdh_bits.js
+@@ -65,15 +65,17 @@ function define_tests() {
+                 });
+             }, namedCurve + " short result");
+ 
++            // // TODO: once we have support of lengths that are not a multiple of 8 bits, uncomment this test
++            // https://github.com/grafana/xk6-webcrypto/issues/80
+             // Non-multiple of 8
+-            promise_test(function(test) {
+-                return subtle.deriveBits({name: "ECDH", public: publicKeys[namedCurve]}, privateKeys[namedCurve], 8 * sizes[namedCurve] - 11)
+-                .then(function(derivation) {
+-                    assert_true(equalBuffers(derivation, derivations[namedCurve], 8 * sizes[namedCurve] - 11), "Derived correct bits");
+-                }, function(err) {
+-                    assert_unreached("deriveBits failed with error " + err.name + ": " + err.message);
+-                });
+-            }, namedCurve + " non-multiple of 8 bits");
++            // promise_test(function(test) {
++            //     return subtle.deriveBits({name: "ECDH", public: publicKeys[namedCurve]}, privateKeys[namedCurve], 8 * sizes[namedCurve] - 11)
++            //     .then(function(derivation) {
++            //         assert_true(equalBuffers(derivation, derivations[namedCurve], 8 * sizes[namedCurve] - 11), "Derived correct bits");
++            //     }, function(err) {
++            //         assert_unreached("deriveBits failed with error " + err.name + ": " + err.message);
++            //     });
++            // }, namedCurve + " non-multiple of 8 bits");
```

Comment:
Should it be removed if not needed?

Reply:
It's needed, it's commented because comments easier to uncomment once support will be there